.azure-pipelines/templates/build/publish-artefacts.yml

```diff
@@ -16,5 +16,6 @@ steps:
     TargetFolder: '$(Build.ArtifactStagingDirectory)/'
 
 - publish: $(Build.ArtifactStagingDirectory)
+  condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
I don't think we need to run the copyfiles task either if we're not running this one, maybe duplicate the condition?